.github/workflows/rust.yaml

```diff
@@ -4,10 +4,12 @@ on:
   push:
     paths-ignore:
       - '*.md'
+      - '**/*.md'
```

Comment:
Is this necessary? I though above entry already covers it.

Reply:
I agree, I'm pretty confident the above one is sufficient

Reply:
If either of them worked, we wouldn't have CI running all of the tests for all commits in this PR

Reply:
I don't agree. 
This file is changed, which is not a `.md` -> `.github/workflows/rust.yaml`
So it is normal to trigger the CI imo

Reply:
Yes, but if you look at the first commit, you'll see that CI was running there as well, while only root markdown file has changed.

Reply:
I see... That is really weird, looking for alternate solutions atm.

Reply:
It seems the correct way to do it is: `**.md`
https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#example-including-paths
Addin a commit now